ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -173,6 +176,7 @@ private IEnumerable<NotificationSite> CreateNotificationSitesFromModel(int notif
             }
         }
 
+        // TODO: Check whether this is actually used by anything!
```

Comment:
I don't think we actually use this dynamic validation anywhere as there are no input field properties on the ClinicalDetails page (apart from othersite, but that uses the NotificationSite validation)